core/src/main/scala/cats/Eval.scala

```diff
@@ -310,6 +310,30 @@ private[cats] trait EvalInstances extends EvalInstances0 {
       }
     }
 
+  implicit val catsReducibleForEval: Reducible[Eval] =
```

Comment:
Couldn't a fairly trivial `Traverse[Eval]` be implemented where `traverse` looks something like this?

```scala
def traverse[G[_]: Applicative, A, B](fa: Eval[A])(f: A => G[B]): G[Eval[B]] =
  f(fa.value).map(Eval.now)
```

It's a bit wonky, but the `Comonad`, `Reducible`, etc instances are already based on eager calls to `.value`, so I don't know if it's any different.